common/xfer/plugin_spec.go

```diff
@@ -7,8 +7,8 @@ import (
 	"sort"
 
 	"github.com/davecgh/go-spew/spew"
-	"github.com/mndrix/ps"
 	"github.com/ugorji/go/codec"
+	"github.com/weaveworks/ps"
```

Comment:
The library hasn't been updated for 4 years, I doubt there's any interest in upstreaming the changes, although I could ask. In the meantime I think we should be using our fork explicitly.